tests/unit/test_task6.py

```diff
@@ -0,0 +1,117 @@
+from unittest.mock import patch, mock_open
+from io import StringIO
+import pytest
+from task6.rosyk_task6 import numbers_sum, odd_even_write, python_posibilities_print, python_change, \
+    guests_greeting, the_counter, text_format, chapters_write, small_big_letters, imdb_rating
+
+
+@pytest.mark.parametrize('input_param, expected', [('1\n2\n3', ('6.0',)), ('0', ('0.0',))])
+def test_find_sum_numbers(input_param, expected):
+    with patch('builtins.open', mock_open(read_data=input_param)):
+        numbers_sum()
+        assert open.mock_calls[5][1] == expected
+
+
+@pytest.mark.parametrize('input_param, expected', [('string', 'file contains something except numbers')])
+def test_find_sum_number_string(input_param, expected, capsys):
+    with patch('builtins.open', mock_open(read_data=input_param)):
+        numbers_sum()
+        captured = capsys.readouterr()
+        assert captured.out.strip() == expected
+
+
+@pytest.mark.parametrize('input_param, expected', [(2, ('2 is even',)), (3, ('3 is odd',))])
+def test_odd_even_write(input_param, expected):
+    with patch('sys.stdin', StringIO(str(input_param))):
```

Comment:
here you need to mock your custom function. not stdin

Reply:
it doesn`t work( It says 'OSError: pytest: reading from stdin while output is captured!' but i don`t understand where. That`s why i mock stdin